tests/robot-tests/tests/libs/utilities.py

```diff
@@ -141,9 +143,23 @@ def set_cookie_from_json(cookie_json):
     sl.driver.add_cookie(cookie_dict)
 
 
-def get_datetime(strf, days=0):
-    now = datetime.datetime.now() + datetime.timedelta(days=days)
-    return now.strftime(strf).lstrip('0')
+def format_uk_to_local_datetime(uk_local_datetime: str, strf: str) -> str:
+    if os.name == 'nt':
+        strf = strf.replace('%-', '%m')
```

Comment:
This is failing on Windows. I think we could replace `%-d` with `%#d` and the same with `m` to make it work - but I'm not sure if that would work on Mac.

Reply:
The failure when running manage_content.robot
```
Add release note to release                                           [ WARN ] Capturing a screenshot at URL https://localhost:5021/publication/aa766e3c-b0b1-49f8-59b8-08d84995cf88/release/c2cec50b-e887-4dda-4701-08d849954a47/content
| FAIL |
Element 'css:#releaseNotes li:nth-of-type(1) time' did not get text '08d August 2020' in 30 seconds.
```

Reply:
Good catch, this was just a typo unfortunately! It's actually supposed to be `%#`. Think it 'should' work on Mac as it's derived from Linux, but I think we could probably just cross that bridge when someone actually runs the tests on Mac.